keentools_facebuilder/blender_independent_packages/pykeentools_loader/install.py

```diff
@@ -20,18 +20,29 @@
 import os
 import sys
 from threading import Thread, Lock
+from enum import Enum
+import bpy
 
 from .config import *
 
 
-__all__ = ['is_installed', 'uninstall', 'installation_status',
+__all__ = ['PartInstallation', 'is_installed', 'uninstall', 'installation_status',
            'install_from_download', 'install_from_download_async',
-           'install_from_file', 'loaded', 'module']
+           'install_core_from_file', 'install_addon_from_file',
+           'download_file_name', 'download_zip', 'download_zip_async',
+           'updates_downloaded', 'loaded', 'module', 'remove_download',
+           'install_download', 'download_file_version_path',
+           'downloaded_keentools_version']
```

Comment:
IMO too many function for a small functionality added. Some functions may be removed/joined/simplified. E.g. why do I need to have `download_file_version_path` in public interface?